README.md

```diff
@@ -4,7 +4,6 @@
 [![Build status](https://ci.appveyor.com/api/projects/status/valslkp8sr50eepn/branch/master?svg=true)](https://ci.appveyor.com/project/Gonum/gonum/branch/master)
 [![codecov.io](https://codecov.io/gh/gonum/gonum/branch/master/graph/badge.svg)](https://codecov.io/gh/gonum/gonum)
 [![go.dev reference](https://pkg.go.dev/badge/gonum.org/v1/gonum)](https://pkg.go.dev/gonum.org/v1/gonum)
-[![GoDoc](https://godoc.org/gonum.org/v1/gonum?status.svg)](https://godoc.org/gonum.org/v1/gonum)
```

Comment:
Resurrect this as a godocs.io link.

Reply:
Shouldn't then all of the package README's that used to link to godoc.org also now link to godocs.io instead of pkg.go.dev? Or should they have both badges, just like here?

Reply:
I think that battle was lost a long time ago. By putting the link to godocs.io on the front page we give people the option to avoid pkg.go.dev if they want to. I'd say that most people do not use the old-repo level READMEs to direct their doc journey so it's less important to provide the link there. If you want to do the work, I'm happy with that, it just didn't seem as important to me. Thanks.

Reply:
Done.

Reply:
Sorry for the delay.

Reply:
All's fine, things happen.